sourcegraph/about-replatform
# Add dev tools quiz

Closes #87 by adding dev tools quiz page

## To Test
- Check out the `/dev-tools-quiz` page and make sure appearance/ behavior match what's on prod on desktop/ mobile.
- Check static builds.
